packages/client/screens/UserMusicPlaylistEditorSearchScreen/__tests__/UserMusicPlaylistEditorSearchScreen.test.tsx

```diff
@@ -0,0 +1,762 @@
+                        },
+                    },
+                    on: {
+                        'Start interacting with the application': {
+                            target: "#Other User MPE Rooms.Displaying user's MPE rooms",
+                        },
+                    },
+                },
+                "Displaying user's MPE rooms": {
+                    initial: 'More MPE rooms to load',
+                    states: {
+                        'More MPE rooms to load': {
+                            always: {
+                                cond: "Has fetched all user's MPE rooms",
+                                target: "#Other User MPE Rooms.Displaying user's MPE rooms.Loaded all MPE rooms",
+                            },
+                            meta: {
+                                test: async (
+                                    { screen }: TestingContext,
+                                    {
+                                        context: { page, searchQuery },
+                                    }: OtherUserMpeRoomsMachineState,
+                                ) => {
+                                    await waitFor(() => {
+                                        const totalMpeRoomsFetched =
+                                            page *
+                                            OTHER_USER_MPE_ROOMS_PAGE_LENGTH;
+                                        const allDisplayedMpeRooms =
+                                            withinOtherUserMpeRoomsScreen(
+                                                screen,
+                                            ).getAllByTestId(
+                                                /^mpe-room-card-/i,
+                                            );
+                                        expect(
+                                            allDisplayedMpeRooms.length,
+                                        ).toBe(totalMpeRoomsFetched);
+                                    });
+
+                                    for (const mpeRoom of getPage(
+                                        filterMpeRoomSummaryByName(
+                                            OTHER_USER_MPE_ROOMS,
+                                            searchQuery,
+                                        ),
+                                        page,
+                                        OTHER_USER_MPE_ROOMS_PAGE_LENGTH,
+                                    )) {
+                                        expect(
+                                            await withinOtherUserMpeRoomsScreen(
+                                                screen,
+                                            ).findByText(mpeRoom.roomName),
+                                        ).toBeTruthy();
+                                    }
+                                },
+                            },
+                        },
+                        'Loaded all MPE rooms': {
```

Comment:
**question:** Unless I'm mistaken, pressing on a mpe card should also be tested isn't it ? We would expect to at least show the default mpeRoom view screen ( by default I mean room not found but would still attest the redirection )

Reply:
Indeed, it's an important part of the model, it should be tested 👍